orttraining/orttraining/python/training/ortmodule/ortmodule.py

```diff
@@ -269,3 +285,36 @@ def named_modules(self, *args, **kwargs):
         """Override :meth:`~torch.nn.Module.named_modules`"""
 
         yield from self._torch_module.named_modules(*args, **kwargs)
+
+    def __getattr__(self, name: str):
+        if '_is_initialized' in self.__dict__ and self.__dict__['_is_initialized'] == True:
+            # If ORTModule is intitialized and attribute is not found in ORTModule,
+            # it must be present in the user's torch.nn.Module. Forward the call to
+            # the user's model.
+            assert '_torch_module' in self.__dict__, "ORTModule does not have a reference to the user's model"
+            return getattr(self.module, name)
+        else:
+            super(ORTModule, self).__getattr__(name)
```

Comment:
```suggestion
            return super(ORTModule, self).__getattr__(name)
```
missed `return` here?

Reply:
Yeah, let me add a return.

Reply:
> Yeah, let me add a return.

is there a easy way to add a test to guard this path?